# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/shadps4/system-ffmpeg.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/CMakeLists.txt	2026-10-14 20:29:19.361042324 +0200
+++ b/CMakeLists.txt	2026-10-14 20:29:19.379713610 +0200
@@ -231,6 +231,11 @@ if(NOT (GIT_REMOTE_URL_LOWER MATCHES "sh
     set(ENABLE_UPDATER OFF)
 endif()
 
+# prefer system FFmpeg over downloading prebuilts
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+find_package(FFmpeg 5.1.2 MODULE)
+list(POP_BACK CMAKE_MODULE_PATH)
+
 if (ENABLE_SYSTEM_LIBRARIES)
     list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
     find_package(Boost 1.84.0 CONFIG)
